tools/Dockerfile

```diff
@@ -11,6 +11,9 @@ RUN cd /tmp \
   && mkdir -p /tmp/build/.modcache \
   && cd /tmp/tools \
   && go install -trimpath -tags=tools github.com/golangci/golangci-lint/cmd/golangci-lint \
-  && chmod -R 777 /tmp/build/
+  && chmod -R 777 /tmp/build/ \
+  && git config --global --add safe.directory /build 
```

Comment:
Is it possible to remove this line now that go has been updated?

Reply:
I thought yes, but no, it is not possible.

Reply:
Can you try if removing the `-buster` from the docker image changes anything?

Reply:
:dart: